src/components/common/animation-control/playback-controls.tsx

```diff
@@ -26,7 +26,7 @@ import {Button, Tooltip, ButtonProps} from 'components/common/styled-components'
 import AnimationSpeedSliderFactory from './animation-speed-slider';
 import {Reset, Play, Pause, Rocket, AnchorWindow, FreeWindow} from 'components/common/icons';
 import {ANIMATION_WINDOW} from '@kepler.gl/constants';
-import {preciseRound} from 'utils/data-utils';
+import {preciseRound} from '@kepler.gl/layers';
```

Comment:
why is this moved to `layers`? I don't see any layer using it. it is heavily used by components